version.py

```diff
@@ -1 +1 @@
-__version__ = 'v2.1.1'
+__version__ = 'v2.2.0'
```

Comment:
https://github.com/nasa/cumulus-message-adapter-python/blob/4f92c4ccab401911b101ad866335a2f9fe5cd62c/example/requirements.txt#L2
Update to match